Sources/SKTestSupport/SkipUnless.swift

```diff
@@ -288,7 +288,12 @@ package actor SkipUnless {
         HoverRequest(textDocument: TextDocumentIdentifier(uri), position: positions["1️⃣"])
       )
       let hover = try XCTUnwrap(response, file: file, line: line)
-      XCTAssertNil(hover.range, file: file, line: line)
+      XCTAssertEqual(
+        hover.range,
+        .init(line: 2, utf16index: 6) ..< .init(line: 2, utf16index: 12),
```

Comment:
I would just use `Position` here instead of `.init` because it’s more explicit about what it’s construing. It also means we don’t need spaces around `..<`. Same in the other tests.

```suggestion
        Position(line: 2, utf16index: 6)..<Position(line: 2, utf16index: 12),
```